test/application_system_test_case.rb

```
@@ -3,5 +3,5 @@
 SimpleCov.command_name 'test:system'
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :headless_chrome, screen_size: [1400, 1400]
+  driven_by :selenium, using: ENV.fetch("SYSTEM_TEST_BROWSER", "headless_chrome").to_sym, screen_size: [1400, 1400]
```

is this giving a default value of chrome?

---

Yep, if the environment variable is missing it'll default to `headless_chrome`

---

Would it be better to just fail if the browser isn't present? Else it would fail silently and run the chrome tests twice, right?

---

Ah, I see what you mean. That's a good point, but equally I want a sensible default so that I don't have to remember to specify the target browser every time I run the system tests on my laptop...

---

i don't feel that strongly about it.